pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -81,14 +75,26 @@ class Config11 {
                         openj9:  'xlc13&&aix715'
                 ],
                 test                : 'default',
+                configureArgs       : [
+                        "openj9"      : '--with-mixedrefs'
+                ],
+                additionalFileNameTag: [
+                        "openj9"    : "mixedrefs"
+                ],
                 cleanWorkspaceAfterBuild: true
         ],
 
         s390xLinux    : [
                 os                  : 'linux',
                 arch                : 's390x',
                 test                : 'default',
-                configureArgs       : '--enable-dtrace=auto'
+                additionalFileNameTag: [
+                        "openj9"    : "mixedrefs"
+                ],
+                configureArgs       : [
+                        "openj9"      : '--enable-dtrace=auto --with-mixedrefs'
+                        "hotspot"     : '--enable-dtrace=auto',
```

Comment:
@karianna Looks like this wasn't fixed by your commit

Reply:
comma moved.